app/controllers/v1/projects_controller.rb

```diff
@@ -22,7 +22,7 @@ def index
       param :path, :id, :string, :required, 'User Id'
     end
     def show
-      project = Project.find_by params[:id]
+      project = Project.find_by id: params[:id]
```

Comment:
I changed these as I was getting an exception viewing Projects, I believe `find_by` takes a field to search